### ISSUE TITLE
Data Balancing

### ISSUE THREAD

**Issue by yuval-livne** [Jacob-Link/DS_LAB_HW2#6]
Make sure that your classes are balanced. If not, the model could become biased towards the class with more samples. If the classes in your dataset are imbalanced, you could use techniques like oversampling the minority class or undersampling the majority class to balance them.

also make sure the data in the training set and in the validation set are preaty much balance
